modules/shared/components/molecules/PostViewHeader/UserInfo/Index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Divider from '../../../atoms/Divider/Divider';
+import type { ReactElement, FC } from 'react';
+import VerticalThreeDotsIcon from '../../../icons/verticalThreeDots.svg';
+import type { PostViewHeaderTypes } from './PostViewHeaderTypes';
+import UserInfo from '../../UserInfo/Index';
+import * as EDivider from '../../../atoms/Divider/types/EDivider';
+
+const PostViewHeader: FC<PostViewHeaderTypes.IProps> = ({
+  size = 'medium',
+  variant,
+  imgSrc,
+  name,
+  date,
+  onClick,
```

Comment:
please be more explicit. `onClick`-ing what exactly?
I would suggest `onClickOption` or something along the lines.